lib/src/ruby/module.cc

```diff
@@ -946,8 +946,13 @@ namespace facter { namespace ruby {
 
         // Expand the command only if expand is true,
         std::string expanded;
-        try{
+        try {
+#ifdef HAS_LTH_EXPAND
             expanded = expand_command(command, leatherman::util::environment::search_paths(), expand);
+#else
+            expanded = expand_command(command);
+            LOG_DEBUG("This version of Leatherman cannot expand shell builtins. Falling back to the default behavior.");
```

Comment:
We could point that is the Leatherman version used while compiling, just an update of Leatherman will not help.